audit/base/base_test.go

```diff
@@ -329,16 +335,18 @@ func TestReset(t *testing.T) {
 		newConn conn.Audit
 		err     bool
 	}{
-		{
-			name:    "Nil client",
-			newConn: conn.NewNoOP(),
-			err:     true,
-		},
-		{
-			name:   "Nil connection",
-			client: must(),
-			err:    true,
-		},
+		/*
```

Comment:
either:
1. Delete this.
2. Uncomment it
3. Leave a reason why it's commented out?

Reply:
Should be uncommented, removed them to test one that was failing by itself.